packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -339,11 +339,41 @@ const takeScreenshot = (Cypress, state, screenshotConfig, options: TakeScreensho
       return null
     })
     .then(() => {
+      // could fail if iframe is cross-origin, so fail gracefully
+      try {
```

Comment:
Does moving this logic here impact event execution at all on the event manager for other screenshot handlers (like in `runner-ct`)? It doesn't look to but wanted to throw a comment out just in case

Reply:
No, the other event manger screenshot handlers don't deal with blackout or animation.